invent-code-0/main.cpp

```diff
@@ -3,5 +3,9 @@
 using namespace std;
 
 int main() {
-  // Write code here.
+  cout<<"I love driving and long drives.\n";
```

Comment:
Nice job using the \n! But to stay consistent can you either use cout << endl everywhere or use \n?

Reply:
Done.